### PR TITLE
Features/137005159/add cancel order option fix

### PR DESCRIPTION
**Pull request by victor-enogwe**
### What does this PR do?
The PR fixes the cancel order feature bugs.
### Description of Task to be completed?
Allow customers to specify cancellation reason selecting from default reasons in a select dropdown
Allow  order admins to see order status on order list page.
Allow order admins to see order comments for cancelled orders .
### What are the relevant pivotal tracker stories?
#137005159 Add cancel order option

### Screenshots (if appropriate)
oc

### CODE REVIEW

File: imports/plugins/core/orders/client/templates/orders.js [hunk 1]
```diff
@@ -298,7 +298,12 @@ Template.orderStatusDetail.helpers({
   orderAge: function () {
     return moment(this.createdAt).fromNow();
   },
-
+  orderStatus: function () {
```
Comment (andela-aolaniran) [inline imports/plugins/core/orders/client/templates/orders.js:301]: Can we use a consistent function declaration style

**Review by andela-aolaniran — COMMENTED** [review]
PR looks good, would checkout the app on heroku.

File: imports/plugins/core/orders/client/templates/orders.js (hunk 1 above)
Comment (victor-enogwe) [reply imports/plugins/core/orders/client/templates/orders.js:301]: some of those functions exists in the app, I used es6 to declare my own functions.

Comment (vonvick) [reply imports/plugins/core/orders/client/templates/orders.js:301]: I agree with @andela-aolaniran

**Review by vonvick — APPROVED** [review]
LGTM. Will need to review the app on heroku

File: imports/plugins/core/orders/client/templates/orders.js (hunk 1 above)
Comment (andela-aolaniran) [reply imports/plugins/core/orders/client/templates/orders.js:301]: Alright @andela-venogwe  fair enough.

**Pull request merged**